packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -2,17 +2,18 @@ import { Type, validators } from '@openmrs/esm-framework';
 
 export const configSchema = {
   search: {
-    // TODO: Rename this config property
-    patientResultUrl: {
+    patientChartUrl: {
```

Comment:
This config property key change is a **BREAKING change** for implementers' config schemas. It will be communicated appropriately in the release notes when the next release it cut.